universal-application-tool-0.0.1/app/auth/FakeAdminClient.java

```diff
@@ -13,11 +13,16 @@
 
   public static final String CLIENT_NAME = "FakeAdminClient";
 
-  public static final ImmutableSet<String> ACCEPTED_LOCALHOSTS =
-      ImmutableSet.of("localhost", "civiform");
+  public static final ImmutableSet<String> ACCEPTED_HOSTS =
+      ImmutableSet.of("localhost", "civiform", "staging.seattle.civiform.com");
 
   private ProfileFactory profileFactory;
 
+  public static boolean canEnable(String host) {
+    return FakeAdminClient.ACCEPTED_HOSTS.stream()
```

Comment:
(somewhat related comment) TIL `ImmutableSet` uses `AbstractCollection.contains` and just iterates.